### PR TITLE
denoting legacy remarks

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Legacy Remarks

Show it and if editable it can be removed, but you can’t re-add it. If you can edit it then ` (legacy)` should let you know that you should remove/replace it. 
Glossary already filters out the legacy ones, so we should be good with preventing the adding of any new legacy ones.

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
auto-formatting bloat I have to address

File: talentmap_api/fsbid/services/common.py [hunk 1]
```diff
@@ -861,11 +862,15 @@ def parse_agenda_remarks(remarks = []):
                         continue
 
             remark['remarkRefData'][0]['rmrktext'] = refRemarkText
+            if remark['remarkRefData'][0]['rmrkactiveind'] == 'N':
+                remark['remarkRefData'][0]['rmrktext'] = '(legacy) ' + remark['remarkRefData'][0]['rmrktext']
```
Comment (scott062) [inline talentmap_api/fsbid/services/common.py:866]: I like it at the beginning but i'm not picky - but either way we capitalize it?

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/common.py:866]: like `(Legacy) ` or `(LEGacy) ` or something else?

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/common.py:866]: hehe

Comment (scott062) [reply talentmap_api/fsbid/services/common.py:866]: sorry meant `Legacy`

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/common.py]: Done

**Review by scott062 — APPROVED** [review]

**Pull request merged**